src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -28,6 +28,12 @@ enum USER_AUTH_CONDITIONS {
   SUSPENDED is provided when the User is currently under an active suspension.
   """
   SUSPENDED
+
+  """
+  DELETION is provided when the User is scheduled for deletion or actively
```

Comment:
kinda picky but it feels like this should be an adjective instead of a noun to fit with the other conditions. Like `PENDING_DELETION` or something?